src/McRefCommon.cpp

```diff
@@ -115,7 +110,7 @@ bool hasNextEvent(EventChain chain, int event) {
  *  if nodeId is right below a migration event, returns the source population of the oldest such migration event.
  *  Otherwise, returns defaultLcaPop
  */
-int migLcaPop(int nodeId, int gen, int defaultLcaPop) {
+int migLcaPop(int nodeId, int gen, int defaultLcaPop) { // TODO - refactor to utilize getMigNodeAbove
```

Comment:
It would be nice to do that. Maybe change getMigNodeAbove so that if the age it gets is negative then it returns the oldest migration node (with largest age).